### PR TITLE
fix: grant role event log ci

### PR DESCRIPTION
**Pull request by viraj124**
- adds absolute path in ci for uploading the event logs json file

### CODE REVIEW

File: packages/solidity-contracts/scripts/hardhat/eventFilter.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { task } from 'hardhat/config';
 import { HardhatRuntimeEnvironment } from 'hardhat/types';
 import { writeFileSync } from 'fs';
+import * as path from 'path';
```
Comment (DefiCake) [inline packages/solidity-contracts/scripts/hardhat/eventFilter.ts:4]: Is this needed?

File: .changeset/dirty-hornets-watch.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@fuel-bridge/solidity-contracts': minor
+---
+
+add absolute path for the event logs in ci
```
Comment (DefiCake) [inline .changeset/dirty-hornets-watch.md:5]: nit, the path does not look "relative" to me in the CI

File: packages/solidity-contracts/scripts/hardhat/eventFilter.ts (hunk 1 above)
Comment (viraj124) [reply packages/solidity-contracts/scripts/hardhat/eventFilter.ts]: removed was testing something

File: .changeset/dirty-hornets-watch.md (hunk 2 above)
Comment (DefiCake) [reply .changeset/dirty-hornets-watch.md:5]: I meant absolute in my previous comment.

`path: ./packages/solidity-contracts/grantedRoles.json` is a relative path. Absolute paths look like `/path/to/file`,  in other words, they do not start with a `.`

What you did is fix the path as it was relative to the root of the project, whereas before it was being treated as relative to the solidity-contracts directory. The path is still relative :)

**Review by DefiCake — DISMISSED** [review]

File: .changeset/dirty-hornets-watch.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@fuel-bridge/solidity-contracts': minor
```
Comment (DefiCake) [inline .changeset/dirty-hornets-watch.md:2]:
```suggestion
'@fuel-bridge/solidity-contracts': patch
```

**Review by PraneshASP — APPROVED** [review]

**Pull request merged**